src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-max-event-buffer-time.mdx

```diff
@@ -29,7 +29,8 @@ Compatible with all agent versions.
 
 ## Description
 
-Sets the maximum time (in seconds) that the agent will store events in memory. When the oldest event exceeds this time, the agent transmits the buffered content at the [harvest cycle](/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle). You must set this value after the start call.
+This method sets the maximum time in seconds that the New Relic iOS agent will store events in memory before including the buffered event to be sent to New Relic in the next [harvest cycle] (/docs/accounts-partnerships/education/getting-started-new-relic/glossary#harvest-cycle).
+In other words, when the oldest event timestamp exceeds this custom configured time, the agent transmits the buffered content at the harvest cycle between each connection from a New Relic agent to the collector. You must set this value after the start call.
```

Comment:
"_after the start call_" . Reads a little funny, perhaps change to "_after the "_Agent.start()_ call" (or insert iOS -specific method name)

Reply:
Will do. Checking with Matt.